### PR TITLE
sortTestSteps

### PR DESCRIPTION
**Pull request by s2oBCN**
- Sort problem with tests steps in TCR and TCP documents.

### CODE REVIEW

**Review by clemensutschig — APPROVED** [review]

**Comment by clemensutschig** [comment]
@s2oBCN Cool. Master fwd Port?

**Comment by s2oBCN** [comment]
> @s2oBCN Cool. Master fwd Port?

Yes, I was waiting for this approval, before the other PR :)

**Pull request merged**
